source/wifi/wifi_hal.c

```diff
@@ -3181,7 +3203,6 @@ INT wifi_getAssociatedDeviceDetail(INT apIndex, INT devIndex, wifi_device_t *out
     nl_cb_set(nl.cb,NL_CB_VALID,NL_CB_CUSTOM,AssoDevInfo_callback,&info);
     nl_recvmsgs(nl.socket, nl.cb);
     nlmsg_free(msg);
-    nl_cb_put(nl.cb);
```

Comment:
why this line is remove?

Reply:
nl_cb_put  has been  handled in nlfree() function, so removing the extra line